edx-platform/pearson-learninghub-theme/lms/static/sass/partials/lms/theme/_login-register.scss

```diff
@@ -1,46 +1,49 @@
-div.login-register-content {
-  background: $background-color-form-login-register;
+.signin-container {
+    box-shadow: 2px 2px 2px $shadow-light-gray;
+    padding: 50px 30px 60px;
 }
 
-.login-register .login-provider {
-  width: auto !important;
-  padding-right: 10px !important;
-  min-width: 165px;
+.signing-title {
+    text-align: center;
+    font-size: 18px;
+    font-weight: 600;
+    margin-bottom: 10px;
 }
 
-.login-register {
-  background: $background-color-form-login-register;
+.login-register .form-wrapper .login-providers {
+    display: flex;
+    text-align: left;
+    background: $guyabano-gray;
```

Comment:
guyabano? it is a color? @luismorenolopera Just asking...

Reply:
Based on this page yes, https://encycolorpedia.com/f8f8f8

Reply:
Haha really weird, but...